pkg/runtime/runtime.go

```diff
@@ -1704,6 +1700,12 @@ func (a *DaprRuntime) startReadingFromBindings() error {
 	}
 	for name, binding := range a.inputBindings {
 		go func(name string, binding bindings.InputBinding) {
+			subscribed := a.isAppSubscribedToBinding(name)
+			if !subscribed {
```

Comment:
```suggestion
			if !a.isAppSubscribedToBinding(name) {
```

Reply:
I didn't see `subscribed` used anywhere, and `!a.isAppSubscribedToBinding(name)` is as readable.